src/Amazon.Lambda.Tools/LambdaDefinedCommandOptions.cs

```diff
@@ -417,5 +417,29 @@ public static class LambdaDefinedCommandOptions
                 ValueType = CommandOption.CommandOptionValueType.StringValue,
                 Description = "Docker image name and tag in the 'name:tag' format."
             };
+        public static readonly CommandOption ARGUMENT_EPHEMERAL_STORAGE_SIZE =
+            new CommandOption
+            {
+                Name = "Ephemerals Storage Size",
+                Switch = "--ephemerals-storage-size",
+                ValueType = CommandOption.CommandOptionValueType.IntValue,
+                Description = "The size of the function's /tmp directory in MB."
+            };
+        public static readonly CommandOption ARGUMENT_FUNCTION_URL_ENABLE =
+            new CommandOption
+            {
+                Name = "Function Url Enable",
+                Switch = "--function-url-enable",
+                ValueType = CommandOption.CommandOptionValueType.BoolValue,
+                Description = "Enable function URL."
```

Comment:
```suggestion
                Description = "Enable function URL. A function URL is a dedicated HTTP(S) endpoint for your Lambda function. When enabled, Lambda automatically generates a unique URL endpoint for you. Function URL endpoints have the following format: https://<url-id>.lambda-url.<region>.on.aws"
```

From https://docs.aws.amazon.com/lambda/latest/dg/lambda-urls.html

Reply:
I added the `A function URL is a dedicated HTTP(S) endpoint for your Lambda function.` part. If I make this too long the formatting looks really bad when users do `dotnet lambda deploy-function --help`.